captum/optim/_param/image/images.py

```diff
@@ -478,23 +478,38 @@ class NaturalImage(ImageParameterization):
     or rescaling to [0,255], it can perform those steps with the provided transforms or
     inside its computation.
     For example, our GoogleNet factory function has a `transform_input=True` argument.
+
+    Arguments:
+        size (list of int): The height and width to use for the nn.Parameter tensor.
+        channels (list of int): The number of channels to use when creating the
+            nn.Parameter tensor.
+        batch (list of int): The number of channels to use when creating the
+            nn.Parameter tensor, or stacking init images.
+        parameterization (ImageParameterization): An image parameterization class.

```

Comment:
I think you mean that `decorrelation_module` is missing 'optional' as parameterization is always required.

Reply:
I made `decorrelation_module` a parameter, because ToRGB is specific to RGB and RGBA images. If someone has a model that uses a different colorspace, then they can add their own color decorrelation module.

Reply:
@NarineK Ah, okay! For some reason I was thinking of the Optional type hint instead of adding optional to the argument docs.